### DISCUSSION TITLE
Show task completion summary in card UI in projects Board layout

### DISCUSSION THREAD

**Discussion by jithunnair-amd** [community/community#52498] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

We have been using the new Projects for our sprints. We create stories as Github issues and add them to the project. We use the Board layout to represent a Kanban board. For each story, we usually add tasks in the description. These tasks show up in the UI on the Github issue page as shown below.
![image](https://user-images.githubusercontent.com/37884920/231082360-5fe0faf8-99af-49d3-99ac-a776f3983fdb.png)

However, often, as part of our scrums, we find that stories will get moved to Done/Accepted even though the tasks in the corresponding Github Issues were not marked as completed (by checking the box). This can lead to "dangling tasks". This could be more easily caught by updating the card UI in the Board layout to show an icon indicating the task completion status eg. "2/2" or something. Is this something that can be done?

**Comment by foysal619** [comment]
I'm glad to hear that you're using the Projects feature in GitHub to manage your sprints and organize your work! It's great to see that you're using issues to represent stories and tasks.

Regarding the tasks that you're adding to the description of each issue, it's important to note that these are just plain text and not actual task lists. GitHub does support task lists, which are a great way to track progress on individual issues and collaborate with your team. Here's how you can create a task list in an issue:

Open an issue and click on the Edit button.

In the issue description, create a list of tasks using the following syntax:

 Task 1
 Task 2
 Task 3
Make sure to include the square brackets and the space between the brackets and the task description.

Save the changes to the issue.
Once you've created a task list, you can check off individual tasks as they are completed by clicking on the checkbox next to the task description. This will add an "X" to the checkbox and mark the task as completed.

In addition to task lists, GitHub also supports other features that can help you manage your sprints, such as labels, milestones, and assignees. These can be used to track the status of individual issues and to coordinate work among team members.

I hope this information is helpful! Let me know if you have any other questions or if there's anything else I can do to assist you.

**Reply by PaulDevenneyWarmur** [reply]
Please remove this response as it is wrong and almost certainly AI generated.

**Comment by jithunnair-amd** [comment]
@foysal619 I'm a bit confused, because I think what I describe and show in the picture is exactly the task list you describe. As I said, task lists are great to mark progress on individual tasks of an issue, but my question was about whether the overall task completion status (eg. 3/5 tasks) can be reflected in the cards for the containing issue on the Kanban board layout.

**Reply by gymdesk-nr** [reply]
@jithunnair-amd If you didn't find your way there already, check out this issue: https://github.com/orgs/community/discussions/16721

**Comment by nawlbergs** [comment]
Lol... I thik [foysal619](https://github.com/foysal619) just pasted a ChatGPT response.

**Comment by marinskeh** [comment]
So... it's not possible yet? To to show progress in checked / unchecken tasks in the preview of an issue?
